castor.php

```diff
@@ -105,3 +107,20 @@ function migrate(): void
     // docker_compose_run('bin/console doctrine:database:create --if-not-exists');
     // docker_compose_run('bin/console doctrine:migration:migrate -n --allow-no-migration --all-or-nothing');
 }
+
+#[AsTask(description: 'Loads fixtures', namespace: 'app:db', aliases: ['fixture'])]
+function fixtures(): void
+{
+    // io()->title('Loads fixtures');
+
+    // docker_compose_run('bin/console doctrine:fixture:load -n');
+}
+
+
+#[AsTask(description: 'Connect to the PostgreSQL database', name: 'db:client', aliases: ['postgres'])]
```

Comment:
```suggestion
#[AsTask(description: 'Connect to the PostgreSQL database', name: 'db:client', aliases: ['postgres', 'pg'])]
```

Team lazy :raised_back_of_hand: :grin:

Reply:
Done ✅